package.json

```diff
@@ -37,7 +37,7 @@
     "jquery": "^3.3.1",
     "knex": "^0.20.6",
     "lodash": "^4.17.15",
-    "mocha": "^7.0.1",
+    "mocha": "^7.2.0",
```

Comment:
I'd remove this, already updated in my branch

Reply:
👍